pgmpy/tests/test_models/test_LinearGaussianBayesianNetwork.py

```diff
@@ -195,5 +195,38 @@ def test_get_random_cpds(self):
         cpds = model_lin.get_random_cpds()
         self.assertEqual(len(cpds), len(model.nodes()))
 
+    def test_get_random(self):
+        model1 = LinearGaussianBayesianNetwork.get_random(n_nodes=5, edge_prob=0.5)
+        model2 = LinearGaussianBayesianNetwork.get_random(n_nodes=5, edge_prob=0.2)
+        self.assertNotEqual(model1.edges(), model2.edges())
+        self.assertIsInstance(
+            model1, LinearGaussianBayesianNetwork, "Incorrect instance"
+        )
+        self.assertIsInstance(
+            model2, LinearGaussianBayesianNetwork, "Incorrect instance"
+        )
+
+        node_names = ["a", "aa", "aaa", "aaaa", "aaaaa"]
+        model3 = LinearGaussianBayesianNetwork.get_random(
+            n_nodes=5, edge_prob=0.5, node_names=node_names
+        )
+        self.assertEqual(len(model3.nodes()), 5)
+        self.assertEqual(sorted(model3.nodes()), node_names)
+        self.assertEqual(len(model3.cpds), 5)
+        self.assertIsInstance(
+            model3, LinearGaussianBayesianNetwork, "Incorrect instance"
+        )
+
+        seed = 42
+        model4 = LinearGaussianBayesianNetwork.get_random(n_nodes=5, seed=seed)
```

Comment:
Hmm, I don't think this would be testable for now. I think you remove the check for now. The DAG.get_random method does not have an argument to specify the seed right now. Because of that model4 and model5 have different structures, resulting in different CPDs. Can you create an issue for adding a seed argument to `DAG.get_random` method?

Reply:
Sure i will create an issue for that. Should i remove checks for model4 and model5 and update the PR?

Reply:
Yes.